### DISCUSSION TITLE
Erro BE ADVISED

### DISCUSSION THREAD

**Discussion by DiegoFlu** [kubilus1/autoortho#394] [General]
Friends, I'm having this error every time I try to load the scenario with auto ortho running. I have Dokan installed and I'm following all the steps, however, I always get this error.
![image](https://github.com/kubilus1/autoortho/assets/73845336/df126c6f-3abd-4b4f-b994-37e7875c3719)
[Log.txt](https://github.com/kubilus1/autoortho/files/12665396/Log.txt)

**Comment by marloscaida** [comment]
Instrucciones prácticas para la instalación de AutoOrtho para Windows 10/11.

Muchas personas no pueden usar AutoOrtho porque no han hecho una buena instalación o no han entendido las instrucciones. Este es un intento más de darles una pequeña ayuda para que esta maravillosa aplicación, les funcione adecuadamente y puedan disfrutarla.

Debe asegurarse que su sistema operativo sea la versión de 64 bits
Funciona en X-Plane 11 o 12

Si no tiene una buena máquina, con buena placa de video y discos veloces, ni se moleste en leer el instructivo. AutoOrtho definitivamente no es para usted. Si la tiene, felicitaciones. Siga estos pasos. Si algo no entiende o no se anima a hacer, pídale ayuda algún amigo que sepa un poco más de Windoes y X-Plane.

1.- Obtenga cualquiera de los siguientes productos:

- Dokan
1.1. Descargue e instale la última versión de Dokan aca:
https://github.com/dokan-dev/dokany/tags
Use DokanSetup.exe o Dokan_x64.msi 

- WinFSP
1.2. Descargue e instale la última versión de WinFSP acá:
https://github.com/winfsp/winfsp/tree/v2.0
(Recomendamos bajar el archivo MSI para su instalación)

2.- Haga la instalación de uno de los dos productos de manera estándar o por defecto, sin cambiar nada durante la instalación.

(Si no funciona en su sistema Dokan, desinstálelo e intente con WinFSP.  Debe reiniciar el equipo, cada vez que instale o desinstale)

En mi caso, no me funcionó DOKAN, pero a mis amigos sí. 
Instalé WinFSP y funcionó (no tengo idea cual es la razón). 
Estas librerías sirven para la creación de una carpeta virtual que solo existe mientras funciona AO.

3.- Obtenga la última versión de AutoOrtho acá:

https://github.com/kubilus1/autoortho/tags

Utilice la versión AutoOrtho_win_xxx.zip (instalación manual)

4.- Descompacte el archivo zipeado  en el lugar que usted desee (puede ser en su disco principal, puede ser dentro de la raíz de X-Plane, etc)

5.- Ubique el único archivo ejecutable en la carpeta de AutoOrtho y cree un acceso directo en su escritorio.

6.- Cree una carpeta con un nombre similar a AutoOrthoCache en un disco con buen espacio y velocidad de acceso. Ahí se va a cargar  toda la cache de imágenes necesaria. Esto es muy importante. Capacidad y velocidad de acceso.

7.- Ejecute, desde el acceso directo del escritorio, el AO.  Este proceso lo va a llevar, en primer lugar a la pantalla de configuración que cuenta con dos pestañas. En la primera pestaña  [SETUP] debe consignar los siguientes datos:

a. La carpeta en donde se encuentra su carpeta de escenarios.  

Normalmente en:
C:\X-Plane 12\Custom Scenery o
C:\Program Files (x86)\Steam\steamapps\common\X-Plane 12 
b. La carpeta donde se van a guardar las imágenes cache que necesita AO y mencionada en el punto 6.

c. La carpeta temporal para la instalación (creo que la trae por defecto). Normalmente se encuentra en:

C:\Users\<Nombre de usuario>\.autoortho-data\downloads

d. La Fuente de las imágenes que va a utilizar (se sugiere utilizar BI que es BING) pero puede intentar con alguna otra.

e. La máxima capacidad que va a contener la carpeta de cache de imágenes mencionada en el punto 6 (se sugiere no menor a 100 Mb). Esto depende de su espacio en disco. Mientras más imágenes pueda guardar la caché, mejor performance va a tener su sistema.

NO SE OLVIDE DE PRESIONAR EL BOTON [SAVE] antes de pasar a la siguiente pestaña.

8. Vaya a la pestaña [SCENERY] y seleccione el área que va a alojar AutoOrtho. Se sugiere empezar con el continente donde usted vuela normalmente. Esto se hace utilizando el botón [INSTALL].

La instalación de los escenarios de AutoOrtho tarda bastante y depende de la velocidad de acceso a Internet. Sea paciente y espere que termine. 
Por ejemplo, en el caso de South America, son más de 8.700 Mb. de archivos.

Una vez que termine la instalación puede presionar el botón [RUN].

9.- En este punto, se van a generar un mínimo de 3 carpetas en Custom Scenery. Estas son las siguientes:

yAutoOrtho_Overlays
z_autoortho

y una tercera carpeta virtual que va a existir solamente mientras esté en ejecución AO.

Esta carpeta puede tener un nombre como “z_ao_sa” y supongo que si usted usa otro contienen se llamará “z_ao_na” (North America), “z_ao_au” (Australia), etc.
La verdad es que no lo sé ya que tengo instalado un solo escenario.

10.- Este proceso termina cuando usted visualice en la pantalla de ejecución de comandos el mensaje que se han terminado de descargar todos los archivos necesarios (esta pantalla está presente mientras AO se está ejecutando).

11.- Lance X-Plane y espere que termine de cargar, con el avión y aeropuerto de su preferencia. No importa cual. Asegúrese que su aeropuerto esté dentro del continente elegido. No se sorprenda si nota que ya no está ese aeropuerto o le faltan cosas. No es un problema.
Con el simulador abierto, busque y copie en otra carpeta el archivo scenery_pack.ini que se encuentra en la carpeta Custom Scenery  (sugerimos en la carpeta DESCARGAS o DOCUMENTOS). Cierre el simulador, sin cerrar AO.

12.- Edite el archivo scenery_pack.ini,  desde conde lo copio en el punto anterior  (esto lo debe hacer con el bloc de notas). Ubique las líneas:

SCENERY_PACK Custom Scenery/yAutoOrtho_Overlays/
SCENERY_PACK Custom Scenery/z_ao_sa/
SCENERY_PACK Custom Scenery/z_autoortho/

Póngalas, en el mismo orden que figura más arriba pero al final del archivo scenery_pack.ini y guarde los cambios.

Parado sobre el archivo scenery_pack.ini, use el botón derecho del mouse para que aparezca el menú contextual, elija la opción PROPIEDADES del archivo, y marque el casillero de solo lectura. 

La idea es que este archivo, a partir de ahora, no pueda ser accesible para realizar cambios, de ninguna manera, por parte de X-Plane. Solo usted podrá alterarlo para agregar algún escenario.
ESTO ES MUY IMPORTANTE.

Recuerde el orden en que van los escenarios en scenery_pack.ini:

I
1000 Version
SCENERY

Aquí van aplicaciones como SAM:

SCENERY_PACK C:\Program Files (x86)\Steam\steamapps\common\X-Plane 12\Resources\plugins\SAM\lib\SAM_GlobalTrees/
SCENERY_PACK Custom Scenery/SAM_DeveloperPack/

A partir de acá todas las carpetas que contengan escenarios. Por ejemplo:

SCENERY_PACK Custom Scenery/Aerosoft - EDLP Paderborn-Lippstadt/
SCENERY_PACK Custom Scenery/Aerosoft - LFMN Nice Cote d Azur X/

A partir de acá y al final de los escenarios los landmarks:

SCENERY_PACK Custom Scenery/X-Plane Landmarks - Berlin and Frankfurt/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Budapest/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Chicago/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Dubai/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Las Vegas/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - London/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Los Angeles/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - New York/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Paris/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Portland/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Rio De Janeiro/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Saint Louis/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - San Francisco/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Sydney/
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Washington DC/

Después los escenarios propios del simulador (ejemplo de X-plane 12:

SCENERY_PACK *GLOBAL_AIRPORTS*

(en X-Plane 11 figura de otra manera)

Luego, todas las librerías necesarias para que funcionen los escenarios personalizados como:

SCENERY_PACK Custom Scenery/ NAPS_library/
SCENERY_PACK Custom Scenery/ OpenSceneryX/

Y finalmente deberían ir las entradas para AO:

SCENERY_PACK Custom Scenery/yAutoOrtho_Overlays/
SCENERY_PACK Custom Scenery/z_ao_sa/
SCENERY_PACK Custom Scenery/z_autoortho/

13.- Copie el archive scenery_pack.ini en la carpeta [CUSTOM SCENER] del simulador. Windows le va a preguntar si quiere que el viejo archivo se sobreescriba con el nuevo. ESTO ES IMPORTANTE. Verifique que scenery_pack.ini sea DE SOLO LECTURA.

14.- Arranque nuevamente el simulador. Le va a aparecer una ventanita con una advertencia, informándole que no puede tener acceso al archivo scenery_pack.ini (eso ya usted lo sabe porque así lo preparamos). 

Si no aparece esta advertencia, comience nuevamente desde el punto 11 de este instructivo.

Si, por el contrario aparece, quiere decir que ha llegado bien hasta este punto. 

Ahora va a tardar un poco más en abrir el simulador, ya que AO está bajando las imágenes y haciendo su trabajo. No se desespere. Tenga paciencia. 

Si su máquina tiene buena capacidad de procesamiento y discos veloces, podrá disfrutar de X-Plane y AO.

Consejo final: Antes de volar de un punto a otro, posicione primero su aeronave en el punto A. Luego, haga lo mismo con el punto B. Una vez finalizado estos procesos, vuele desde A a B.
Tardé toda una mañana en “actualizar” con AO todos mis aeropuertos preferidos. Espero les sea de utilidad.

Ojalá sirva.

**Comment by DiegoFlu** [comment]
Friend, thank you very much for the tips. I removed dokan and installed WinFSP and now it works perfectly. I don't know how to thank you, it helped a lot! Thanks

**Comment by zippy77777** [comment]
Niether dokan or winfsp work for me
